src/DashJS/SDK/Platform/methods/documents/create.ts

```diff
@@ -4,6 +4,15 @@ declare interface createOpts {
     [name:string]: any;
 }
 
+/**
+ * Create documents on the platform
```

Comment:
Minor suggestion - This might be more accurate since nothing actually goes to the platform until the `broadcast`.
```suggestion
 * Create and prepare documents for the platform
```

Reply:
Addressed in https://github.com/dashevo/DashJS/pull/31